can-construct.js

```diff
@@ -663,6 +709,40 @@ assign(Construct, {
 		 * console.log(Counter.count); // 1
 		 * ```
 		 */
+	},
+	/**
+	 * @function can-construct.ReturnValue ReturnValue
+	 * @parent can-construct.static
+	 * 
+	 * A constructor function which `value` property will be used as a value for a new instance. 
+	 * 
+	 * @signature `new Construct.ReturnValue( value )`
+	 * 
```

Comment:
The signature should talk about how this works.  Also, the example code below should be put in the signature.